### PR TITLE
portfolio and watchlist functionality

### PR DESCRIPTION
**Pull request by smarquezmgh**
- redux state added into portfolio component and watchlist component
- to many changes with enums call and enums files
- multiple changes in portfolio components(UI) for functional integration

### CODE REVIEW

**Review by nanSer0 — CHANGES_REQUESTED** [review]
This branch has conflicts that must be resolved

**Review by nanSer0 — APPROVED** [review]

**Comment by OswaldUnity** [comment]
This branch has conflicts that must be resolved

File: src/ui/common/landsMenu.ui.tsx [hunk 1]
```diff
@@ -1,55 +1,59 @@
...
-            ))}
+
+            }
+            )}
           </>
           :
           <>
-            {(Object.keys(Metaverses) as Array<keyof typeof Metaverses>).map((key) => {
-              if (Metaverses[key] === Metaverses.ALL) {
+            {(Object.keys(MetaverseOptions) as Array<keyof typeof MetaverseOptions>).map((key) => {
```
Comment (OswaldUnity) [inline src/ui/common/landsMenu.ui.tsx]: Last time you created a type for this

File: src/ui/common/landsMenu.ui.tsx [hunk 2]
```diff
@@ -1,55 +1,59 @@
...
 export default function LandsMenuUI({ metaverse, setMetaverse, form, isBorder }: LandsMenuUIProps) {
-
   return (
     <div className={`w-full flex items-center justify-center  py-16 ${isBorder ? "border-b border-nm-remark" : ""}`}>
       {
         form == ButtonForm.Horizontal ?
           <>
-            {(Object.keys(Metaverses) as Array<keyof typeof Metaverses>).map((key) => (
-              <button
+            {typedKeys(MetaverseOptions).map((key) => {
```
Comment (OswaldUnity) [inline src/ui/common/landsMenu.ui.tsx]: Last time you created a type for this

**Review by nanSer0 — CHANGES_REQUESTED** [review]
adjust comments and conflicts

**Pull request merged**